modApiServer/src/org/aion/api/server/ApiTxResponse.java

```diff
@@ -78,12 +78,18 @@ public String getMessage() {
                 return "Transaction successfully repaid";
             case ALREADY_SEALED:
                 return "Transaction has already been sealed in the repo";
-            case REPAYTX_POOL_EXCEPTION:
-                return "Repaid transaction wasn't found in the pool";
+            case REPAYTX_BUFFER_FULL:
+                return "Repaid transaction dropped due to the buffer full";
```

Comment:
Could we add the strings to the `TxResponse` class? It allows reuse and this code can be simplified to directly return the message associated with the issue instead of the `switch` statement.